tools/scripts/generate-new-piece.ts

```diff
@@ -49,8 +48,8 @@ const generateIndexTsFile = async (pieceName: string) => {
     })
     .join('')
 
-  const indexTemplate = `
-import { createPiece } from "@activepieces/pieces-framework";
+  const indexTemplate =
+`import { PieceAuth, createPiece } from "@activepieces/pieces-framework";
```

Comment:
Please edit the one in `create action documentation` as well if it's required.,

Reply:
done => [2456af4](https://github.com/activepieces/activepieces/pull/1635/commits/2456af4aeb18e0e8c61023a1122cb1eefeb9385b)